### DISCUSSION TITLE
Wheel / Pie Menu for adding nodes

### DISCUSSION THREAD

**Discussion by Correct-Syntax** [GimelStudio/GimelStudio#50] [Ideas]
WIP idea for an intuitive menu to add nodes.

**Reference**

![Screenshot_2021-04-22_07 12 40](https://user-images.githubusercontent.com/60711001/115736663-0ff00a00-a351-11eb-8aa6-1d76c135cd71.png)
![Screenshot_2021-04-22_084348](https://user-images.githubusercontent.com/60711001/115737107-7543fb00-a351-11eb-9075-abffa29133c1.png)
![GS_-_UI_Idea_04](https://user-images.githubusercontent.com/60711001/115736669-12526400-a351-11eb-85db-e6e76bda4d46.png)
![Combine](https://user-images.githubusercontent.com/60711001/115736680-14b4be00-a351-11eb-9c02-4bf89cc57114.png)

**Comment by jackcarey** [comment]
[NodeRED](https://nodered.org/) uses a collapsible sidebar for it's library list (see left below). The use of section headings and colour on each node make it easy to navigate. With GimelStudio already having panels that can be (un)docked, this could be an easier approach to take? Pie menus are good for a small amount of options but my experience is that they are unwieldy with a large number of text items, especially as power users could have a lot of nodes as GimelStudio develops.

![NodeRED screenshot](https://user-images.githubusercontent.com/4663918/63022233-76304400-be70-11e9-8516-cab988df6b1e.png)
